journeyman/webdev/markdown.md

```diff
@@ -0,0 +1,26 @@
+# Markdown 
+
+## Intro
+
+Markdown is a text-to-html tool for web developers. It allows you to write information that is easy write, and easy to write than it converts it to an html structure for you.
```

Comment:
I'm going to have you fix this intro. A few suggestions:

* Make it so there are no spelling or grammatical errors.
* Provide context for why you might want to use markdown
  * It works great for ReadMes and documentation
  * You can write long documents with it
  * You can use it to format your text in a lot of popular modern chat apps, like Discord and Slack
  * You can use it in Github comments, issues, and pull requests (In fact, Github has it's own special flavor of Markdown - you should mention that too!)